src/common/pool_map.c

```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright 2016-2021 Intel Corporation.
+ * (C) Copyright 2016-2022 Intel Corporation.
```

Comment:
File didn't change?

Reply:
This happens if there is a merge conflict and the file has changed this year